themes/ddbasic/scripts/flexmenu.js

```diff
@@ -0,0 +1,181 @@
+/* Flexmenu plugin with B14 changes	
+  

```

Comment:
We've modified the plugin to work for our usecase, which is why it's not placed in the contrib folder.